Launcher/QuantConnect.Lean.Launcher.csproj

```diff
@@ -74,6 +74,10 @@
       <Project>{39a81c16-a1e8-425e-a8f2-1433adb80228}</Project>
       <Name>QuantConnect.Algorithm.CSharp</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Algorithm.FSharp\QuantConnect.Algorithm.FSharp.fsproj">
```

Comment:
I _think_ think was not linked in due to a dll loading issue with the composer. We'll need to do some testing to ensure its working as expected

Reply:
The 'QuantConnect.Algorithm.FSharp.dll' is not copied to 'Launcher otherwise and therefore cannot test FSharp algos just by changing config.json. I did not see any issues in win/mac/Linux builds/algo tests. 


Previous the F# project target runtime was not 4.5.2 so that was causing issues as well. It is now 4.5.2 to match the rest.

If you do decide not to include it, then we will need additional instructions in F# readme.md to ensure users are not tripped up when running sample F# algorithms.

Reply:
Another option for F# is to provide a post-build event to copy the dll to the launcher's bin directory